src/etcpal/inet.c

```diff
@@ -456,31 +456,25 @@ bool etcpal_ip_network_portions_equal(const EtcPalIpAddr* ip1, const EtcPalIpAdd
  */
 
 /*!
- * \fn etcpal_error_t etcpal_inet_ntop(const EtcPalIpAddr* src, char* dest, size_t size)
+ * \fn etcpal_error_t etcpal_ip_to_string(const EtcPalIpAddr* src, char* dest)
  * \brief Convert IPv4 and IPv6 addresses from binary to text form.
```

Comment:
I'd still say something like "This works similarly to a BSD api's inet_ntop()".  That way anyone doing a quick search for ntop could find the replacement function.

Reply:
I have this on line 462:
```
 * This function uses each platform's inet_ntop() function under the hood.
```
Is that sufficient?

Reply:
You can also see it rendered in the staging docs: https://etclabs.github.io/EtcPal/docs/stage/29/group__etcpal__inet.html#gaf98da0ae21f7960b5e046c55b8d79c4a

Reply:
I didn't see that.  Yes, that's good enough.  Thanks!